### ISSUE TITLE
1.2020.7.3 - Object Reference Error when filtering on SystemUserId.

### ISSUE THREAD

**Issue by mtnmuncher** [rappen/FetchXMLBuilder#347]
In the condition below, when I select systemuserid, I receive attached error.

![fetchxmlbuilder_error](https://user-images.githubusercontent.com/6696112/88397776-70d2bd80-cd92-11ea-8ef5-4385d02f39cf.png)

```
<fetch top="50" >
  <entity name="contact" >
    <link-entity name="systemuser" from="systemuserid" to="owninguser" >
      <filter>
        <condition/>
      </filter>
    </link-entity>
  </entity>
</fetch>
```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Cinteros.Xrm.FetchXmlBuilder.Controls.conditionControl.RefreshOperators()
   at System.EventHandler.Invoke(Object sender, EventArgs e)
   at System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   at System.Windows.Forms.ComboBox.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by rappen** [comment]
Thanks for reporting @mtnmuncher - this is most likely the same issue as was reported in #345 and will be addressed for next release.

**Comment by rappen** [comment]
Fix is now released in version 1.2020.9.1 🙂
